### ISSUE TITLE
double `stylelint-disable` throw error

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#874]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

When we use `stylelint-disable` double, we get error.

> Which rule, if any, is this issue related to?

no matter

> What CSS is needed to reproduce this issue?

```scss
.test {
    display: inline;
    /* stylelint-disable rule-no-duplicate-properties */
    *display: inline;
    /* stylelint-disable rule-no-duplicate-properties */
}
```

> What stylelint configuration is needed to reproduce this issue?

no matter

> Which version of stylelint are you using?

e.g. `4.5.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

no matter

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

No error.

> What actually happened (e.g. what warnings or errors you are getting)?

```shell
CssSyntaxError: stylelint: /path/to/_test.scss:4:5: A new disabled range cannot begin until the previous one has ended
```

**Comment by davidtheclark** [comment]
This is the intended behavior. It was a bug before that it did *not* throw this error. Are you intending to use `stylelint-enable`?

```css
.test {
    display: inline;
    /* stylelint-disable rule-no-duplicate-properties */
    *display: inline;
    /* stylelint-enable rule-no-duplicate-properties */
}
```

**Comment by evilebottnawi** [comment]
@davidtheclark I get error if use
```scss
.test {
    display: inline;
    /* stylelint-disable rule-no-duplicate-properties */
    *display: inline;
    /* stylelint-disable color-named */
   color: red;
}
```
I disable rule cascade, i think we don't get error.

**Comment by davidtheclark** [comment]
@evilebottnawi Are you using the latest version or the other one? If that's all the CSS you have and you are using the latest version of stylelint *and* you are getting an error, it might be a bug.

**Comment by evilebottnawi** [comment]
@davidtheclark latest, 4.5.0 version.

**Comment by davidtheclark** [comment]
What is the error then?

**Comment by evilebottnawi** [comment]
@davidtheclark 
```shell
CssSyntaxError: stylelint: /path/to/_test.scss:4:5: A new disabled range cannot begin until the previous one has ended
```
All info how i write above.

**Comment by davidtheclark** [comment]
Oh whoops: #864 is not merged yet. Once that is merged that you are doing will be possible. Until then, the error is correct --- and it was a bug that it was not thrown before.

**Comment by davidtheclark** [comment]
To clarify: you cannot have overlapping disabled ranges.

(After #864 is merged and released, you will be able to.)

Thanks for the quick communication to clarify.
